go/worker/compute/executor/tests/tester.go

```diff
@@ -148,8 +164,4 @@ blockLoop:
 			t.Fatalf("failed to receive block")
 		}
 	}
-
-	// Requeuing same call should fail.
```

Comment:
Is this tested somewhere else?

Reply:
Hm probably, but not explicitly/directly. I'll add the test back (just need to change the check a bit)